tests/end2end/playwright/atlas.spec.js

```diff
@@ -0,0 +1,183 @@
+// @ts-check
+import { test } from '@playwright/test';
+import { AtlasPage } from './pages/atlaspage';
+import { expectParametersToContain } from './globals';
+
+test.describe('Atlas', () => {
```

Comment:
```suggestion
test.describe('Atlas', () => {
```

New tests mut be tagged `@readonly`

Reply:
Fixed